.github/workflows/node.js.yml

```diff
@@ -26,6 +26,9 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
+    - name: Install Dependencies 1    
     - run: npm ci
+    - name: Install Dependencies 2
+    - run: npm ci --prefix frontend/package-lock.json
```

Comment:
fixed in MR: https://github.com/EternalDoritos/cinema-booking-system/commit/774bb52e471841e2482d07209f8c1dd5eef8ff97